sdk/advisor/Azure.ResourceManager.Advisor/tests/ScenarioTests/ConfigurationTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Advisor.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Advisor.Tests
+{
+    public class ConfigurationTests : AdvisorManagementTestBase
+    {
+        public ConfigurationTests(bool isAsync)
+                    : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        private ResourceGroupResource ResourceGroup { get; set; }
+
+        private CpuThreshold TestThreshold = CpuThreshold.Twenty;
+        private CpuThreshold DefaultThreshold = CpuThreshold.Five;
+
+        private async Task SetCollection()
+        {
+            ResourceGroup = await CreateResourceGroupAsync();
+        }
+
+        [Test]
+        public async Task ConfigureSubscriptionTest()
+        {
+            string configName = "default";
+            var configData = new ConfigData
+            {
+                Exclude = false,
+                LowCpuThreshold = TestThreshold
+            };
+
+            await DefaultSubscription.CreateInSubscriptionConfigurationAsync(configName, configData);
+            var data = await DefaultSubscription.GetConfigurationsBySubscriptionAsync().ToEnumerableAsync();
+            Assert.AreEqual(TestThreshold, data.FirstOrDefault(x => x.Name.Equals(configName)).LowCpuThreshold);
+            Assert.IsFalse(data.FirstOrDefault(x => x.Name.Equals(configName)).Exclude);
+
+            configData.LowCpuThreshold = DefaultThreshold;
+            await DefaultSubscription.CreateInSubscriptionConfigurationAsync(configName, configData);
+            data = await DefaultSubscription.GetConfigurationsBySubscriptionAsync().ToEnumerableAsync();
+            Assert.AreEqual(DefaultThreshold, data.FirstOrDefault(x => x.Name.Equals(configName)).LowCpuThreshold);
+            Assert.IsFalse(data.FirstOrDefault(x => x.Name.Equals(configName)).Exclude);
+        }
+
+        [Test]
+        public async Task ConfigureResourceGroupTest()
+        {
+            await SetCollection();
```

Comment:
since the implementation of this method is very simple, could we just expand that here and remove the method & property of resourceGroup?

Reply:
Done